### ISSUE TITLE
Home gone from web resource Group after remove from admin resource group

### ISSUE THREAD

**Issue by onepack** [modxbot/migrate#3491]
onepack created Redmine issue ID 3491

I moved Home to Admin resources that I created to test. When I was finished I decided to remove the Home from that resource group but one way or the other it didn't come back to be in the web resource group or it disappeared there too...

I now have no homepage on the website!

It is still available in the database and it still loads when you visit the website it's just gone in the resource tree and in the menu.

I can access it via the url saved url with doc id 1, but saving something results in the "endless save"..

**Comment by onepack** [comment]
onepack submitted:

I found out that in the table modx_site_content table the homepage had it's own id to be it's parent...
I set it back to be 0 (web) and now the Homepage is back again.

This strange setting happened while dragging the resource. I remember I saw Home - Home on the right side before it disappeared.

Maybe adjust the drag functionality so that a document cannot become the child from itself.
